src/test/java/org/apache/commons/io/channels/ByteArraySeekableByteChannelCompressTest.java

```diff
@@ -62,7 +66,7 @@ void testCloseIsIdempotent() throws Exception {
     @ParameterizedTest
     @ValueSource(ints = { 0, 1, 2, 3, 4, 5, 6 })
     void testReadingFromAPositionAfterEndReturnsEOF(final int size) throws Exception {
-        try (SeekableByteChannel c = new ByteArraySeekableByteChannel(size)) {
+        try (SeekableByteChannel c = ByteArraySeekableByteChannel.wrap(new byte[size])) {
```

Comment:
I don't think this is a good idea for reading tests. You want to test reading from the same array over and over when you are reading because you want to make sure there are no side-effects that incorrectly _write_ to the underlying byte array.